mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule BitstylesPhoenix.MixProject do
   def project do
     [
       app: :bitstyles_phoenix,
-      version: "2.3.0",
```

Comment:
Why 6? 
If the goal is to somehow match this to the bitstyles version then that is not needed. One can configure the bitstyles version used (also the default assumption) and there is no need for the versions to align.

Reply:
We also usually bump the version as part of the hex release.